### DISCUSSION TITLE
Is it possible to Integrate mitmweb UI in React JS project without <iframe/> ?

### DISCUSSION THREAD

**Discussion by Ramnirmal0** [mitmproxy/mitmproxy#5052] [Q&A]
Hi, @mhils,

My requirement is, I have to integrate mitmweb UI  into my React JS project without <iframe/>. Is it possible? 

Can you please guide me through this? I am new to mitmproxy.

**Comment by mhils** [comment]
Hi @Ramnirmal0! It's probably a bit tricky, but surely it's possible - all source code for the front end is freely at https://github.com/mitmproxy/mitmproxy/tree/main/web.

The type of integration you seem to be asking for is only really beneficial to your company's product and does not contribute to the mitmproxy project in any way. I'm not doing unpaid consulting work, so I'm afraid you need to figure it out on your own. :)
